tests/core/test_settings.py

```diff
@@ -385,3 +390,21 @@ def test_usage_data_collection_disabled(env_var, conf_setting, is_enabled, clear
     else:
         with conf_patch:
             assert is_usage_data_collection_enabled() == is_enabled
+
+
+@patch("airflow.utils.log.secret_masker.mask_secret")
+@patch("airflow.configuration.conf.get")
+@patch("airflow.configuration.get_section_and_key_for_env")
+@patch("airflow.settings.ENV_VARIABLES_TO_MASK", ["AIRFLOW__CORE__FERNET_KEY", "AIRFLOW__NONMASK_ENV"])
+def test_mask_conf_values(self, mock_get_section_and_key, mock_get, mock_mask_secret):
+    mock_get_section_and_key.side_effect = [
```

Comment:
I don't understand why this needs to be mocked - by mocking it you never test the function you added